docs/what_makes_securedrop_unique.rst

```diff
@@ -0,0 +1,105 @@
+What Makes SecureDrop Unique
+============================
+
+SecureDrop attempts to solve or mitigate several problems journalists and sources
+have faced in recent legal investigations, attacks from state actors, and other
+threats to the confidentiality of communications.
+
+No third parties that can secretly be subpoenaed
+------------------------------------------------
+
+For decades, there were very few leak prosecutions in the United States in large
+part because the government would have to subpoena reporters to testify against
+a source to get a conviction. That proved incredibly difficult, if not impossible,
+when reporters regularly refused to testify and threatened to go to jail rather
+than betray a source.
+
+More recently, there have been a record number of leak prosecutions largely because
+the government has learned they don’t need reporters to testify against their
+sources anymore. Instead, they can just secretly subpoena third party services
+like Google or AT&T or Verizon or Facebook and get a treasure trove of digital
+information on reporters and sources’ communications. For example, the Associated
+Press had twenty of their phone lines subpoenaed without their knowledge in order
+to identify a source. The government also got a warrant for Fox News reporter James
+Rosen’s Gmail account without him knowing. In both cases, their alleged sources
+were prosecuted, even though journalists never directly divulged their sources.
```

Comment:
Hi this is a great summary. I would suggest adding dates to examples contained in here. Starting with "More recently," means that in 18th months this paragraph may be less punctual.

even though "the" journalists never